CMakeLists.txt

```
@@ -1,9 +1,13 @@
-cmake_minimum_required(VERSION 3.18...3.29)
+cmake_minimum_required(VERSION 3.26...3.29)
 project(${SKBUILD_PROJECT_NAME} LANGUAGES C)
 
-find_package(Python REQUIRED COMPONENTS Interpreter Development.Module)
+find_package(Python REQUIRED COMPONENTS Interpreter Development.Module ${SKBUILD_SABI_COMPONENT})
 
-python_add_library(deflate_ext MODULE src/deflate.c)
+if(Python_VERSION VERSION_GREATER_EQUAL 3.11)
+  python_add_library(deflate_ext MODULE USE_SABI 3.11 src/deflate.c)
+else()
+  python_add_library(deflate_ext MODULE src/deflate.c)
```

```suggestion
  python_add_library(deflate_ext MODULE WITH_SOABI src/deflate.c)
```

Oops, forgot this before. You want to have the SOABI on binaries in wheels.